src/search/bookmarks.ts

```diff
@@ -14,6 +14,7 @@ export const addBookmark = (getDb: DBGet, tabManager: TabManager) => async ({
     timestamp?: number
     tabId?: number
 }) => {
+    // url = normalizeUrl(url);
```

Comment:
Is this intentional?

Reply:
Good catch. Left over from refactoring. Not needed. (this whole function is not currently used just serves as a reference of the new style, documented in refactoring.md)